### ISSUE TITLE
GRPC Support

### ISSUE THREAD

**Issue by dbettin** [JasperFx/alba#83]
Is it possible to test ASP.Net Core GRPC services via Alba?

**Comment by jeremydmiller** [comment]
@dbettin Sorry for the delay. I can't think of any reason why that wouldn't work. There's nothing *yet* in Alba to help formulate requests or read responses for GRPC, but I take pull requests:)

**Comment by jeremydmiller** [comment]
@dbettin I did some research today. I don't think that Alba really buys you much here for strictly GRPC endpoints. If you have a service that exposes both GRPC & normal ReST services, then you can just use the `TestServer` instance that hangs off of Alba 3/4's `SystemUnderTest` or v5's `AlbaHost` to create GRPC channels not unlike this post: https://thecloudblog.net/post/integration-tests-for-grpc-services-in-asp.net-core/

Long story short, I don't think Alba would do anything useful for GRPC integration testing and at most, there's a little bit of documentation to show how you'd use Alba's test harness to create GRPC client services.

**Comment by jeremydmiller** [comment]
I'm closing this as I don't think there's anything actionable here.
